### DISCUSSION TITLE
Farming Expectations

### DISCUSSION THREAD

**Discussion by Taresin** [Chia-Network/chia-blockchain#457] [Q&A] [answered]
### Context
I've successfully plotted a single k=32. It's only just finished so the "Total Chia Farmed" value is at 0.

### What can I expect?
Just wanted to get a feel of what kind of time scale to expect this value to change.
Also, what kind of values do we expect this to increment by?

I understand right now Chia in beta and not a lot of people are trying out the chia blockchain so the values would be much higher than once launched in MainNet.
I'd like to find out what cadence to check my dashboard to see if my set up is working properly. (i.e. farm a non-zero Chia value)

### How are rewards earned?
I've read that it will initially reward 16 Chia every 5 minutes.
I'm not exactly sure how the reward gets distributed.
I assume that each Chia is discovered by a pool and members of that pool share the reward according to their plot size.
If this is the case, is there a way to determine the pool size to calculate an expected value?

**Comment by hoffmang9** [comment, marked as answer]
This FAQ should help - https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#how-do-i-tell-if-im-farming-correctly

Because there is a farming contest going on right now, the netspace estimate is 4288TiB. One k=32 is 101GiB. In current consensus there are 16 TXCH awarded to one farmer every 5 minutes. Your chance of winning the next block is .101/4288 or 0.0024%.

New consensus is going to change that. Moving forward 16 farmers will win 1 TXCH every 5 minutes. We will go from 288 chances to win a day (once every 5 minutes) to 4,608 chances to win everyday. That document should come out this week and the first release of new consensus is only a few weeks away.

**Reply by Taresin** [reply]
Oh I see! thanks for adding more context.
Is there any significance to winning TXCH in Beta outside of the competition? I assume that the launch MainNet will reset everything back to zero.

**Reply by hoffmang9** [reply]
TXCH is only for testing and will get reset both at mainnet and probably once or twice more before then also.

**Comment by hoffmang9** [comment]
There will be two pool protocols. One is plots committed to a pool from inception and it does not require on chain fees or permission to set up but does require replot to switch. There will be another using chialisp that will require you to spend a transaction on chain but then will let you pretty quickly (a few hours delay) switch pools. Both will be shortly after mainnet, especially with the new 4608 chances per day change. There is some info about the pool protocol in the new consensus doc that will be posted shortly.
